src/desktop/apps/order2/routes.tsx

```diff
@@ -58,8 +58,14 @@ export const checkoutFlow = async (req, res, next) => {
 
     res.status(status).send(layout)
   } catch (error) {
-    console.log("(apps/order2) Error: ", error)
-    next(error)
+    if (error.message.includes("Received status code 404")) {
```

Comment:
#minor let's move the `console.log` (could be `console.error`) up above the `if` block so we get it  in  both places.

I'm assuming you found this PR: https://github.com/artsy/force/pull/3312 and it seems like we're not entirely sure what the "ideal" solution is just yet.

I do want to fix this so we avoid erroneous 500s being reported, but will also make a ticket (somewhere) to look into the future/better solution. Maybe we're seeing the 500 bubble up somewhere due to an update in our networking  package? (see https://github.com/artsy/reaction/pull/1777). I'm not sure. Let's bring it up at the next  front-end practice meeting at the very least!

Reply:
1. I will move the console.log!
2. Yes, that PR was our "inspiration". None of us feel great about it, but it solves the problem....
3. Great tip on the updated network package. I'll try reverting that locally, to see if it changes functionality. 

Thank you!